ThatValidator.js

```diff
@@ -362,10 +347,10 @@
 
                 for(var i = 0; i < elements.length; i++)
                 {
-                    var field = elements[i];
+                    var field2 = elements[i];
 
                     //if the element has not already been found
-                    if(self.fields.indexOf(field) == -1)
+                    if(self.fields.indexOf(field2) == -1)
```

Comment:
Sigh... Lost in translation. I'll fix it.


Sent from Samsung Mobile

<div>-------- Original message --------</div><div>From: Jonathan Dumaine <notifications@github.com> </div><div>Date:09-14-2015  4:26 PM  (GMT-05:00) </div><div>To: JonDum/ThatValidator <ThatValidator@noreply.github.com> </div><div>Cc: mlohbihler <ml@serotoninsoftware.com> </div><div>Subject: Re: [ThatValidator] Added configurable debounce delay and other small
  stuff (#2) </div><div>
</div>In ThatValidator.js:

>  
>                      //if the element has not already been found
> -                    if(self.fields.indexOf(field) == -1)
> +                    if(self.fields.indexOf(field2) == -1)
I think you did it in the wrong place then. I see what you're talking about in the validate() function, but this is in the init() function

—
Reply to this email directly or view it on GitHub.